packages/gatsby-cli/src/create-cli.js

```diff
@@ -146,6 +146,25 @@ function buildLocalCommands(cli, isLocalSite) {
 
     handler: getCommandHandler(`serve`),
   })
+
+  cli.command({
+    command: `graphiql`,
```

Comment:
`explore`? `api-explorer`? graphiql is more of an implementation detail here, it may be more helpful to pick a command and desc that describe what it does vs what the app is called

Reply:
Good point. I like `explore`, will go with that unless anyone has any other thoughts on it?

Reply:
@jquense I went with `data-explorer` as `explorer` sounded a bit vague for the command description and console.log(). I'm happy to change it though.